app/assets/javascripts/controllers/schedule/schedule_form_controller.js

```diff
@@ -33,17 +32,18 @@ miqAngularApplication.controller('scheduleFormController', ['$http', '$scope', '
       $scope.newRecord                         = true;
       $scope.scheduleModel.action_typ          = 'vm';
       $scope.scheduleModel.filter_typ          = 'all';
-      $scope.scheduleModel.enabled             = '1';
-      $scope.scheduleModel.filterValuesEmpty   = true;
-      var today                                 = new Date();
-      var tomorrowsDate                         = parseInt(today.getDate()) + 1;
-      $scope.scheduleModel.miq_angular_date_1  = today.getMonth() + 1 + "/" + tomorrowsDate + "/" + today.getFullYear();
+      $scope.scheduleModel.enabled             = true;
+      $scope.filterValuesEmpty   = true;
+      var today                                = new Date();
+      var tomorrowsDate                        = parseInt(today.getDate()) + 1;
+      $scope.scheduleModel.start_date          = new Date(today.getMonth() + 1 + "/" + tomorrowsDate + "/" + today.getFullYear());
```

Comment:
Sure, that'd be great :). But the last day of month problem should probably be fixed before that.. I notice in firefox, `new Date('03/32/2015')` gives me 04/01/2015 but in chrome, it returns Invalid Date.